aiogram/dispatcher/filters/builtin.py

```diff
@@ -728,18 +728,20 @@ def __init__(self, chat_type: typing.Container[ChatType]):
 
         self.chat_type: typing.Set[str] = set(chat_type)
 
-    async def check(self, obj: Union[Message, CallbackQuery, ChatMemberUpdated]):
+    async def check(self, obj: Union[Message, CallbackQuery, ChatMemberUpdated, InlineQuery]):
         if isinstance(obj, Message):
-            obj = obj.chat
+            chat_type = obj.chat.type
         elif isinstance(obj, CallbackQuery):
-            obj = obj.message.chat
+            chat_type = obj.message.chat.type
```

Comment:
Callback query can be without `message`, so, we need to add additional condition to prevent to attribute error.